src/Pelagos/Util/DoiRest.ini.example

```diff
@@ -0,0 +1,3 @@
+doi_api_user_name = "username"
+doi_api_password = "password"
+url = "https://website.here"
```

Comment:
new line

Reply:
Now, I feel that separate file for REST API isn't required, only the URL is changed, probably i'll add additional REST URL in the DOIUtil.ini file and consume that.